docs/features/toolbar.md

```diff
@@ -71,6 +71,8 @@ toolbar: {
 
  * **`items`** &ndash; An array of toolbar item names. Most of the components (buttons, dropdowns, etc.) which can be used as toolbar items are described under the {@link features/index Features} tab. A full list is defined in {@link module:ui/componentfactory~ComponentFactory editor.ui.componentFactory} and can be listed using the following snippet: `Array.from( editor.ui.componentFactory.names() )`. Besides button names, you can also use the dedicated separators for toolbar groups (`'|'`) and toolbar lines (`'-'`).
 
+ * **`removeItems`** &ndash; An array of toolbar item names. With this setting you can modify the default toolbar items of a bundle without the need of recreating it from scratch. If, after removing an item, toolbar will have two or more consecutive separators (`'|'`), the duplicates will be removed automatically.
```

Comment:
```suggestion
 * **`removeItems`** &ndash; An array of toolbar item names. With this setting you can modify the default toolbar configuration without the need of defining the entire list (you can specify a couple of buttons that you want to remove instead of specifying all the buttons you want to keep). If, after removing an item, toolbar will have two or more consecutive separators (`'|'`), the duplicates will be removed automatically.
```